.gitignore

```diff
@@ -17,6 +17,7 @@ target/
 *.log
 log-*.txt
 log-*/
+!log-collector/
```

Comment:
What's this needed for?

Reply:
Without this the directory for the new crate gets ignored

Reply:
Ohh the line above, duh. Thanks.